frontend/src/components/pages/CreateReview/CreateReview.tsx

```diff
@@ -78,10 +98,90 @@ const CreateReview = (): React.ReactElement => {
     addBook(book);
   };
 
-  // useEffect hook adds dummy data to the books array
+  /**
+   * This function maps a list of AuthorResponse objects to a list
+   * of Author objects
+   *
+   * @param authorResponses - a list of AuthorResponse objects returned by the review API
+   * @returns a list of Author objects
+   */
+  const mapAuthorResponseToAuthor = (
```

Comment:
This is prolly what I would have done, but what're your thoughts on having DTO types (ie review/response) and entity types (ie `Author`) to be used in component be separate? I get the coupling argument, but seems like we'd have to have a lot of this mapping code everywhere going forward

Reply:
good point - one thing I feel like we could do is do the mapping within the API client itself. Let me know how you want me to proceed for this PR though

Reply:
Yea true, for now could you maybe move this into a seperate util file or something? Might make more sense than leaving it here so other devs can use it

Reply:
done